admin/includes/template/dashboardWidgets/tplWhosOnline.php

```diff
@@ -17,26 +17,31 @@
+  <div class="col-xs-2"><?php echo zen_wo_get_visitor_status_icon(3); ?>&nbsp;&nbsp;<?php echo $widget['users'][3]; ?></div>
 </div>
 <div class="row">
   <div class="col-xs-4"><?php echo WO_GUEST; ?></div>
-  <div class="col-xs-2"><?php echo zen_wo_get_visitor_status_icon(0); ?>&nbsp;&nbsp;<?php echo $tplVars['widget']['guests'][0]; ?></div>
-  <div class="col-xs-2"><?php echo zen_wo_get_visitor_status_icon(1); ?>&nbsp;&nbsp;<?php echo $tplVars['widget']['guests'][1]; ?></div>
-  <div class="col-xs-2"><?php echo zen_wo_get_visitor_status_icon(2); ?>&nbsp;&nbsp;<?php echo $tplVars['widget']['guests'][2]; ?></div>
-  <div class="col-xs-2"><?php echo zen_wo_get_visitor_status_icon(3); ?>&nbsp;&nbsp;<?php echo $tplVars['widget']['guests'][3]; ?></div>
+  <div class="col-xs-2"><?php echo zen_wo_get_visitor_status_icon(0); ?>&nbsp;&nbsp;<?php echo $widget['guests'][0]; ?></div>
+  <div class="col-xs-2"><?php echo zen_wo_get_visitor_status_icon(1); ?>&nbsp;&nbsp;<?php echo $widget['guests'][1]; ?></div>
+  <div class="col-xs-2"><?php echo zen_wo_get_visitor_status_icon(2); ?>&nbsp;&nbsp;<?php echo $widget['guests'][2]; ?></div>
+  <div class="col-xs-2"><?php echo zen_wo_get_visitor_status_icon(3); ?>&nbsp;&nbsp;<?php echo $widget['guests'][3]; ?></div>
 </div>
 <div class="row">
   <div class="col-xs-4"><?php echo WO_SPIDER; ?></div>
-  <div class="col-xs-2"><?php echo zen_wo_get_visitor_status_icon(0); ?>&nbsp;&nbsp;<?php echo $tplVars['widget']['spiders'][0]; ?></div>
-  <div class="col-xs-2"><?php echo zen_wo_get_visitor_status_icon(1); ?>&nbsp;&nbsp;<?php echo $tplVars['widget']['spiders'][1]; ?></div>
-  <div class="col-xs-2"><?php echo zen_wo_get_visitor_status_icon(2); ?>&nbsp;&nbsp;<?php echo $tplVars['widget']['spiders'][2]; ?></div>
-  <div class="col-xs-2"><?php echo zen_wo_get_visitor_status_icon(3); ?>&nbsp;&nbsp;<?php echo $tplVars['widget']['spiders'][3]; ?></div>
+  <div class="col-xs-2"><?php echo zen_wo_get_visitor_status_icon(0); ?>&nbsp;&nbsp;<?php echo $widget['spiders'][0]; ?></div>
+  <div class="col-xs-2"><?php echo zen_wo_get_visitor_status_icon(1); ?>&nbsp;&nbsp;<?php echo $widget['spiders'][1]; ?></div>
+  <div class="col-xs-2"><?php echo zen_wo_get_visitor_status_icon(2); ?>&nbsp;&nbsp;<?php echo $widget['spiders'][2]; ?></div>
+  <div class="col-xs-2"><?php echo zen_wo_get_visitor_status_icon(3); ?>&nbsp;&nbsp;<?php echo $widget['spiders'][3]; ?></div>
 </div>
 <div class="row">
-  <div class="col-xs-6"><?php echo WO_TOTAL; ?> <?php echo $tplVars['widget']['total']; ?></div>
-  <div class="col-xs-6"><?php echo '<a href="' . zen_href_link(FILENAME_WHOS_ONLINE) . '">' . WO_FULL_DETAILS . '</a>'; ?></div>
+  <div class="col-xs-12">&nbsp;</div>
+</div>
+<div class="row">
+  <div class="col-xs-12"><?php echo WO_TOTAL; ?> <?php echo $widget['total']; ?></div>
+</div>
+<div class="row">
+  <div class="col-xs-12"><?php echo '<a href="' . zen_href_link(FILENAME_WHOS_ONLINE) . '">' . WO_FULL_DETAILS . '</a>'; ?></div>
```

Comment:
I think it looks nicer if the summary stats are each on their own line.  If you disagree, I can back this out.